Resources/views/CRUD/dashboard__action.html.twig

```diff
@@ -0,0 +1,4 @@
+<a class="btn btn-link btn-flat" href="{{ action.url }}">
+    <i class="fa fa-{{ action.icon }}"></i>
+    {{ action.label|trans({}, action.translation_domain|default('SonataAdminBundle')) }}
+</a>
```

Comment:
@greg0ire the patch has already been merge. I'll watch out for this next time though